src/scenarios/shared/startup.py

```diff
@@ -53,16 +53,28 @@ def runtests(self, apptorun: str, **kwargs):
             '--app-exe', apptorun,
             '--metric-type', kwargs['startupmetric'], 
             '--scenario-name', "%s - %s" % (kwargs['scenarioname'], kwargs['scenariotypename']),
-            '--trace-file-name', '%s_startup.etl' % kwargs['exename'],
+            '--trace-file-name', '%s_%s_startup.etl' % (kwargs['exename'], kwargs['scenariotypename']),
```

Comment:
this potentially will introduce spaces into the file name. I guess that means everything else already handles spaces? Nice! :)

Reply:
Why will this introduce spaces? I would assume that is scenariotypename is None that it is just ""

Reply:
scenariotypename has space in it, like ```SDK (Clean Build)```

Reply:
Of course